src/main/java/com/google/cloud/spanner/jdbc/SpannerJdbcDialect.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.spanner.jdbc;
```

Comment:
This class should be moved into com.google.cloud.dataproc.dialects package.
If this dialect works for SpannerToGCS template too, then we will get rid of the com.google.cloud.spanner.jdbc package as whole.

Initially this package was create to override the JdbcPreparedStatement class due to dialect issues.